### DISCUSSION TITLE
Release for axios 1.7.5

### DISCUSSION THREAD

**Discussion by gbremond** [HBOCodeLabs/wiremock-captain#531] [General]
Hi @satvik-s ! 
Would it be possible to make a release with the new axios version (> 1.7.3) :) ? 

Because the version included in wiremock-captain 3.3.1 is vulnerable : 
<img width="932" alt="image" src="https://github.com/user-attachments/assets/a99ad584-888c-489f-be10-c7071d8deee6">

Thank you so much !

**Comment by satvik-s** [comment]
Hi!

Sure, I will try to get to it this upcoming week.

**Comment by satvik-s** [comment]
closing in favor of #536
